src/containers/ClientDetailPage.js

```diff
@@ -15,8 +15,12 @@ import {
 import { Link } from 'react-router-dom';
 
 import { MainMenu } from '../MainMenu';
-import FixedMenu from '../fixedMenu';
-import { clientList } from '../clientList';
+import FixedMenu from '../FixedMenu';
+
+// REVIEW: As we approach MVP, we probably don't want to be
```

Comment:
Not for this PR (enough going on already), but in one of the next rounds, how about we just move the 'data' for one client into here or something and get rid of 'DummyClients'?

Reply:
Hmm - I'd rather keep it separate: it makes it easier to pull out the code cleanly when we no longer need it, and importing `dummyClients` is a nice red flag that this isn't production ready.

Reply:
Makes sense.